rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -116,25 +116,44 @@ enum LaneType {
     Shoulder,
 }
 
+#[derive(Default)]
+struct Width {
+    min: Infer<Metre>,
+    target: Infer<Metre>,
```

Comment:
Just so I understand, `target` is our best guess, but we can specify bounds?

Reply:
correct